news/2025/2025-08-25-performance-points-star-rating-updates.md

```diff
@@ -0,0 +1,238 @@
+- A [change](https://github.com/ppy/osu/pull/33004) by [StanR](https://osu.ppy.sh/users/7217455) to handle the Magnetised mod in performance calculations
+- A [change](https://github.com/ppy/osu/pull/34081) by [StanR](https://osu.ppy.sh/users/7217455) to handle the Deflate mod in performance calculations
+- A [refactor](https://github.com/ppy/osu/pull/32410) by [StanR](https://osu.ppy.sh/users/7217455) to aid with maintaining slider-related calculations
+- A [refactor](https://github.com/ppy/osu/pull/33218) by [Natelytle](https://osu.ppy.sh/users/17607667) to improve understanding of deviation calculations
+- A [refactor](https://github.com/ppy/osu/pull/33265) by [tsunyoku](https://osu.ppy.sh/users/11315329) to aid with difficulty calculation maintenance
+- A [refactor](https://github.com/ppy/osu/pull/33423) by [Givy120](https://osu.ppy.sh/users/10560705) to optimise rhythm calculations
+- A [refactor](https://github.com/ppy/osu/pull/33271) by [Givy120](https://osu.ppy.sh/users/10560705) to improve readability of the new Hidden bonus
+
+## osu!taiko
+
+### Steeper accuracy scaling & rhythm penalties
+
+A [change](https://github.com/ppy/osu/pull/34188) by [rloseise](https://osu.ppy.sh/users/6793778) was made to replace existing accuracy calculations with a steeper formula. This means differences in accuracy now have a larger effect on PP.
+
+To reward high accuracy on harder beatmaps, the SR bonus in accuracy PP now diminishes as the estimated unstable rate increases:
+
+![](/wiki/shared/news/2025-08-25-performance-points-star-rating-updates/taiko-accuracy-scaling.png)
+
+Additionally, higher estimated unstable rates will diminish the rhythm difficulty contribution to PP. This addresses scores which sacrifice accuracy in order to ignore hard rhythms and play them as much easier ones. This penalty scales by how much of the beatmap's difficulty is made up by difficult rhythm, meaning beatmaps without much rhythm difficulty are mostly unaffected. In order to balance alongside this change, rhythm as a whole was buffed and rhythmically complex beatmaps will have a higher SR than before.
+
+This is how the rhythm penalty looks on [MAX - Checklist (feat. Chromeo) [123 // abc]](https://osu.ppy.sh/beatmapsets/2155487#taiko/4543374) (a significantly rhythmically complex beatmap):
+
+![](/wiki/shared/news/2025-08-25-performance-points-star-rating-updates/taiko-rhythm-penalty.png)
+
+### Length bonus revamp
+
+A [change](https://github.com/ppy/osu/pull/33582) by [rloseise](https://osu.ppy.sh/users/6793778) was made in order to entirely revamp the length bonus in PP.
+
+Previously, the length bonus was applied based on the number of objects in the map and would reach its maximum bonus at 1500 objects. Length bonus now scales by a weighted object count depending on how consistently difficult the beatmap is, gradually converging towards a maximum bonus rather than ending abruptly:
```

Comment:
```suggestion
Longer maps award more PP, but this effect was capped at 1,500 objects. [rloseise](https://osu.ppy.sh/users/6793778) has [removed this cap and entirely replaced the "length bonus" formula](https://github.com/ppy/osu/pull/33582) as part of this update.

The length bonus is now scaled by how consistently difficult the beatmap is, and it gradually converges towards a maximum rather than ending abruptly:
```